src/app/data/AdData.jsx

```diff
@@ -0,0 +1,229 @@
+import { containsEmail, extractEmail, formatDate, isValidEmail, mailtoInString } from "@/app/_common/utils/utils";
+import DOMPurify from "isomorphic-dompurify";
+import fixLocationName from "@/app/_common/utils/fixLocationName";
+import { parseISO } from "date-fns";
+
+/**
+ *  --------------------------- Ad Data ---------------------------
+ */
+export default function mapAdData(queryData) {
```

Comment:
query er mye brukt ellers på det man sender inn. `?q=Utvikler` osv. Så jeg tenker at denne kanskje burde hete noe annet, for at det ikke blir for mange ting som heter noe med query. Hva med å bare kalle den `adDataFromBackend` eller noe?

Reply:
Eller kanske något i stil med `rawElasticSearchAdResult` för att vara ännu tydligare?

Reply:
fixed